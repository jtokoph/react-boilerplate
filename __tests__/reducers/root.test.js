import createRootReducer from '../../src/reducers/root';

test('createRootReducer should be a function', () => {
  const rootReducer = createRootReducer();
  expect(typeof rootReducer).toBe('function');
});

test('createRootReducer return default state', () => {
  const rootReducer = createRootReducer();
  expect(rootReducer(undefined, {})).toMatchSnapshot();
});
