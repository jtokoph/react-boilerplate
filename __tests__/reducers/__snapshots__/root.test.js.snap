// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createRootReducer return default state 1`] = `
Object {
  "counter": 0,
}
`;
